packages/melos/pubspec.yaml

```diff
@@ -26,6 +26,7 @@ dependencies:
   glob: ^2.1.2
   graphs: ^2.3.1
   http: ^1.1.0
+  intl: ^0.19.0
```

Comment:
Check which version that is pinned to flutter stable, this should not be higher than that one. I think it might be 0.18.3, but double check.

Reply:
Based on these tags of this commit  https://github.com/flutter/flutter/commit/8d54abfc221e947ff387814b9c91b3a152ede269 I would say 0.19.0 intl is integrated in Flutter version 3.22.0